### ISSUE TITLE
Make it possible to see stats of ai in spectatormode

### ISSUE THREAD

**Issue by Monsterfisch** [Sh0wdown/UnofficialCrusaderPatch#620]
Make it possible to see the popularity stats, granary content and stockpile, etc when clicking on the corresponding building.

Buildings could be split up like follows
- church/chapel/cathedral shows religion stats
- barracks show army stats
- houses show population window
- clicking on peasants or the campground in front of the keep shows the popularity window
- fear factor buildings show fear factor
- granary shows granary content 
- stockpile shows stockpile content
- keep shows taxes
- armory shows armory content
- Inn shows the inn specific stats
- apothecary  shows the apothecary specific stats

**Comment by Heroesflorian** [comment]
Btw, when clicking the advisor in the bottom-right corner, a (regular) player can view various stats in the advisor's book menu. 
The advisor's book menu is "read-only", i.e. nothing can directly be changed there iirc, and it exists already. 

So maybe it might be an alternative to just let spectator players click the keep of a player to open that advisor's book menu for that player? 

That wouldn't contain the exact ale coverage % (but still the ale popularity bonus in the popularity tab), and it wouldn't cover apothecary specific stats, but pretty much everything else from your list would be covered there.

**Comment by Monsterfisch** [comment]
fair point but the reason why i would want the specific gui is to see how many % are left for the next popularity boost to take effect.  but yeah if the ability to change settings is an issue restricting it to the adviser book is perfectly fine too

**Comment by Heroesflorian** [comment]
Btw, the result of the formula used for "Balanced" targetting might be an interesting value to display for spectators as well, maybe?
https://github.com/Sh0wdown/UnofficialCrusaderPatch/wiki/Targeting-Type

**Comment by J-T-de** [comment]
There are some third party tools like the [SHCLiveStatReader](https://github.com/patel-nikhil/SHCLiveStatReader) and as far as I know, @PodeCaradox also works on a similar tool (at least he spoilered it in the UCP Discord). These tools can be adapted to see what you want in an overlay. This might be faster then implementing it graphically into the game...

If it is only for "debugging" single ais, Krarilotus Spectator mode enabled maps do precisely what you want for the player, see [Video](https://www.youtube.com/watch?v=XGBPj8KhOzo)

**Comment by GRhin** [comment]
Yes, so shclivestatreader pulls data and saves it in a file, which can be read and displayed wherever you want (I use this on my stream). Podes tool overlays directly onto the game window itself (after making it windowed).
